src/Umbraco.Web/Models/Mapping/ContentModelMapper.cs

```diff
@@ -142,9 +142,33 @@ private static void AfterMap(IContent content, ContentItemDisplay display, IData
                 TabsAndPropertiesResolver.AddListView(display, "content", dataTypeService);
             }
 
+            var currentDocumentType = contentTypeService.GetContentType(display.ContentTypeAlias);
+            var currentDocumentTypeName = currentDocumentType == null ? String.Empty : currentDocumentType.Name;
+            var docTypeValue = String.Empty;
+
+#if (DEBUG)
+            if (canAccessUser)
+            {
+                var currentUser = userService.GetUserById(UmbracoContext.Current.Security.CurrentUser.Id);

```

Comment:
using the userService.GetUserById is not necessary since you already have the current IUser object in`UmbracoContext.Current.Security.CurrentUser`

I am reviewing and modifying this PR so no need to take any actions.